libs/cart/src/reducers/cart/cart.reducer.ts

```diff
@@ -30,10 +30,21 @@ export function cartReducer<T extends DaffCart>(
     case DaffCartActionTypes.CartClearAction:
     case DaffCartActionTypes.AddToCartAction:
     case DaffCartActionTypes.CartCreateAction:
+		case DaffCartItemActionTypes.CartItemUpdateAction:
+		case DaffCartItemActionTypes.CartItemDeleteAction:
```

Comment:
So item update or delete will cause the cart feature slice to be mutating? I don't understand that behavior.

Reply:
All cart related actions cause the cart feature area to be mutating, but they usually do it through their respective feature areas. The cart feature loading derives its state from the other loading states. But because the cart item update and delete actions no longer trigger the cart item feature area to be loading, neither will they trigger the cart feature area to load. So I needed to trigger the cart feature loading manually with these two action streams.

Reply:
Also, I was wrong here. I was under the impression that I was adding to the cart feature reducer, when that reducer doesn't actually exist.